contacts.go

```diff
@@ -11,10 +11,10 @@ type ContactsSvc interface {
 	Create(params *CreateContactRequest) (CreateContactResponse, error)
 	ListWithContext(ctx context.Context, audienceId string) (ListContactsResponse, error)
 	List(audienceId string) (ListContactsResponse, error)
-	GetWithContext(ctx context.Context, audienceId, contactId string) (Contact, error)
-	Get(audienceId, contactId string) (Contact, error)
-	RemoveWithContext(ctx context.Context, audienceId, contactId string) (RemoveContactResponse, error)
-	Remove(audienceId, contactId string) (RemoveContactResponse, error)
+	GetWithContext(ctx context.Context, audienceId, id string) (Contact, error)
```

Comment:
just changed from `contactId` to `id` so it is most clear that this argument can be either an ID or email str

Reply:
the SDK `v3` will fix this by having a proper Params struct type as an argument here.

Reply:
nice